templates/CRM/Mailing/Form/InsertTokens.tpl

```diff
@@ -54,8 +54,26 @@ var isMailing    = false;
 {/if}
 {literal}
 
+/**
+ * Checks if both the Save Template and Update Template fields exist.
+ * These fields will not exist if user does not have the edit message
+ * templates permission.
+ *
+ * @param {String} prefix
+ */
+function manageTemplateFieldsExists(prefix) {
+  var saveTemplate = document.getElementsByName(prefix + "saveTemplate");
```

Comment:
I can see you copied this style from what was already on the form - but it looks like old pre-jquery code & it would be cleaner with jquery

@colemanw do you have any thoughts on the best way to make this change to fit with our more recent code approaches?

Reply:
Sure let me know what I need to change @eileenmcnaughton.

cc @colemanw

Reply:
The underlying thing is we should use jquery. I think if it's not in a closure we still use cj - so ```cj(prefix + "updateTemplate")``` rather than ``` document.getElementsByName(prefix + "updateTemplate");``` but we also use ```$(prefix + "updateTemplate")``` where we can....

ping @colemanw @seamuslee001 who are probably clearer on this.

Reply:
Ping @colemanw , @seamuslee001